__tests__/addresses.test.js

```diff
@@ -0,0 +1,11 @@
+import TestUtils from "./test-utils";
+
+describe("addresses api", () => {
```

Comment:
We should try to create another test here where we set a new gap limit and see that the addresses loaded is greater than this value.

Reply:
Done